Models/Request/SubmitSightingRequest.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace whale_spotting.Models.Request
+{
+    public class SubmitSightingRequest
+    {
+        [Required]
+        
+        public string Species { get; set; }
```

Comment:
do we want to constrain how long some of these fields are just so the user can't put horribly long names/locations etc. in?

Reply:
Constraints have been added